<launch>
    <include file="$(find isse_crazy)/launch/init_swarm.launch"/>

    <include file="$(find vrpn_client_ros)/launch/sample.launch">
    	<arg name="server" value="172.20.37.251" />
  	</include>

    <group ns="/combo/">
	    <node pkg="isse_crazy" type="StartQuadService.py" name="StartQuadService"></node>
	</group>
</launch>
